layers/+misc/multiple-cursors/funcs.el

```diff
@@ -0,0 +1,34 @@
+;; -*- lexical-binding: t -*-
+;;
+;;; funcs.el --- Spacemacs Multiple Cursors Layer packages File
+;;
+;; Copyright (c) 2012-2018 Sylvain Benner & Contributors
+;;
+;; Author: Codruț Constantin Gușoi <codrut.gusoi@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defun spacemacs//evil-mc-paste-transient-state-p ()
```

Comment:
Is the double slash `//` a typo?

Reply:
It is equivalent to `--` which denotes a function is "private" and should not be used in other files, or relied upon to have a stable API, or still exist in the future.